server/middleware/checkPopulateUserCaseloads.ts

```diff
@@ -0,0 +1,63 @@
+import { RequestHandler, Router } from 'express'
+import { validate as uuidValidate } from 'uuid'
+import logger from '../../logger'
+import PrisonApiService from '../services/prisonApi/prisonApiService'
+import CsipApiService from '../services/csipApi/csipApiService'
+import { ServiceConfigInfo } from '../services/csipApi/csipApiClient'
+import { CaseLoad } from '../interfaces/caseLoad'
+
+export default function checkPopulateUserCaseloads(
+  prisonApiService: PrisonApiService,
+  csipApiService: CsipApiService,
+): RequestHandler {
+  const router = Router()
+
+  router.get('/service-not-enabled', (_req, res) => {
+    res.status(200)
+    res.render('service-not-enabled')
+  })
+
+  router.use(async (req, res, next) => {
+    const splitUrl = req.url.split('/').filter(Boolean)
+    if (
```

Comment:
Missed this last review. The population of the case load properties is separate from the check for service switch on. The population will need to be done outside of the if so that they are available for App Insights and other uses